### PR TITLE
challenges solved

### PR DESCRIPTION
**Pull request by whichking**

### CODE REVIEW

File: 2017/07/01_playing_with_time.py [hunk 1]
```diff
@@ -72,6 +74,9 @@ def date_to_string(date_or_datetime: Union[date, datetime]) -> str:
     :param date_or_datetime: a date or datetime object to convert
     """
 
+    date_string = date_or_datetime.isoformat()
+
+    return date_string[:10]
```
Comment (mplanchard) [inline 2017/07/01_playing_with_time.py:79]: Interesting solution here. I would have reached for `strptime()`, but this works too. Note that this *wouldn't* work if we wanted to do non-iso formatted months and dates, without the leading zero for numbers less than ten

File: 2017/07/01_playing_with_time.py [hunk 2]
```diff
@@ -88,6 +95,10 @@ def string_to_date(date_string: str) -> date:
 def is_future(dt: datetime) -> bool:
     """Return whether the passed datetime is in the future"""
 
+    if dt > datetime.now():
+        return True
+    else:
+        return False
```
Comment (mplanchard) [inline 2017/07/01_playing_with_time.py:101]: Because `dt > datetime.now()` returns a boolean, you can also just `return dt > datetime.now()`

**Review by mplanchard — APPROVED** [review]
Tres bien! 🍖

File: 2017/07/01_playing_with_time.py (hunk 1 above)
Comment (whichking) [reply 2017/07/01_playing_with_time.py:79]: This was before I saw the strptime( ) thing in the docs, but, yeah, that's probably the better way.

File: 2017/07/01_playing_with_time.py (hunk 2 above)
Comment (whichking) [reply 2017/07/01_playing_with_time.py:101]: I thought there was probably a one-line solution to this.

**Pull request merged**